packages/app-store/zoomvideo/lib/VideoApiAdapter.ts

```diff
@@ -202,8 +204,69 @@ const zoomAuth = (credential: Credential) => {
   };
 };
 
+type ZoomRecurrence = {
+  end_date_time?: string;
+  type: 1 | 2 | 3;
+  end_times?: number;
+  repeat_interval?: number;
+  weekly_days?: number; // 1-7 Sunday = 1, Saturday = 7
+  monthly_day?: number; // 1-31
+};
+
 const ZoomVideoApiAdapter = (credential: Credential): VideoApiAdapter => {
   const translateEvent = (event: CalendarEvent) => {
+    const getRecurrence = ({
+      recurringEvent,
+      startTime,
+      attendees,
+    }: CalendarEvent): { recurrence: ZoomRecurrence; type: 8 } | undefined => {
+      if (!recurringEvent) {
+        return;
+      }
+
+      let recurrence: ZoomRecurrence;
+
+      switch (recurringEvent.freq) {
+        case Frequency.DAILY:
+          recurrence = {
+            type: 1,
+          };
+          break;
+        case Frequency.WEEKLY:
+          recurrence = {
+            type: 2,
+            weekly_days: dayjs(startTime).tz(attendees[0].timeZone).day() + 1,
```

Comment:
dayjs is 0-6 and zoom is 1-7 so we have to add one, luckily their start date is Sunday as is Dayjs so we're good there.